### PR TITLE
add sources and targets to index and dropdown

### PR DESCRIPTION
**Pull request by sebgoa**
Small stop gap measure to show sources and target documentation.

we need to follow up in subsequent PR to go over all docs for sources and targets because some don't make sense through our console since they are very ko and kubectl centric.

### CODE REVIEW

File: docs/sources/index.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Event sources
 
-The Triggermesh event Sources are still under active development so this document will be evolving rapidly over the next
-months. Feel free to regularly check back for updates on current Targets as well as the addition of new ones!
+The following is a list of Triggermesh event Sources some available as open source projects some available as hosted solutions on our Cloud.
```
Comment (antoineco) [inline docs/sources/index.md:3]:
```suggestion
The following is a list of Triggermesh event Sources, some available as open source projects, some available as hosted solutions on our Cloud.
```

Comment (antoineco) [inline docs/sources/index.md:3]: Shall we use the same phrasing in the targets' index?

Comment (sebgoa) [reply docs/sources/index.md]: yes

**Comment by sebgoa** [comment]
I removed the existing targets and sources docs from the menu, they just don't make sense.

**Pull request merged**